service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -191,7 +191,7 @@ private String formatTitle(int index, Direction direction, List<DirectionConfigu
         return format("%d. %s %s %s", index, direction.getDirectionType(), lowerCase(config.due.toString()),
             ofNullable(direction.getDateToBeCompletedBy())
                 .map(date -> formatLocalDateTimeBaseUsingFormat(date, config.pattern))
-                .orElse("unknown"));
+                .orElse(""));
```

Comment:
this is covered by my story I am working on, not in yours. We should be removing the BLANK - PLEASE COMPLETE entries if they appear and when the sdo is finalised

Reply:
my story said this in it : Where a user has not entered a value in a field, this must display blank and must not display the values of either  "Null", "UNKNOWN" or "Blank" in document so wasn't sure..

will remove this part! :)